src/Resources/config/client.xml

```diff
@@ -5,7 +5,7 @@
            xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
 
     <services>
-        <service id="nexy_slack.client" class="Nexy\Slack\Client" public="true">
+        <service id="Nexy\Slack\Client" class="Nexy\Slack\Client" public="true">
```

Comment:
- The class option is not needed anymore.

```suggestion
        <service id="Nexy\Slack\Client" public="true">
```

- Could you please create a deprecated alias to keep BC??

Reply:
Removing the class attribute will make the tests fail (don't know how to fix them). Deprecating an alias is not allowed so I made a copy of the whole service definition.

Thoughts?

Reply:
In that case, you can do the reverse: Keep the service id on the definition and add an alias with the service class name as done here: https://github.com/sonata-project/SonataAdminBundle/pull/5039/files#diff-ea95afcce69fc04cf8246bfd1d13ae75R14

I don't know if deprecating the service definition will only trigger error while using service ID or also the alias, it has to be tested.

Reply:
I tried that in one of the commits but when updating a project where i'm testing this, it didn't stop the deprecation  notice from firing. The last commit keeps BC while muting the deprecation notice.

Reply:
Thoughts @Soullivaneuh

Reply:
Sorry for the late reply, didn't see the notification.

Please just use an alias, not a big deal if no deprecation error.

Reply:
We still need to remove the class. You may add edit authorization so I could finish the PR.

Reply:
Sorry, I am not used to working on open projects in github and a bit unsure of what you want me to do. Should I resolve the conversation or do you need me to do something else?

Reply:
Doing something else: https://github.com/nexylan/slack-bundle/pull/31#discussion_r227781542

Reply:
Removed class from service. 
Removed class from the has service test
Added has alias test

All checks out now, I hope! Would you mind taking another look at it @soullivaneuh ?